dev/reference/transactions/transaction-isolation.md

```diff
@@ -57,18 +57,13 @@ The MySQL Repeatable Read isolation level is not the snapshot isolation level. T
 
 ## Transaction retry
 
-Transactions that fail may automatically be retried by TiDB, which may lead to lost updates. This feature can be disabled by setting `tidb_retry_limit = 0`.
+Failed transactions might be automatically retried by TiDB, which can lead to update loss. You can enable **this feature** by setting `tidb_disable_txn_auto_retry = 0`.
```

Comment:
```suggestion
By default TiDB will not retry transactions since this may lead to lost updates. If your application can tolerate lost updates, and does not require Snapshot Isolation consistency, you can enable **this feature** by setting `tidb_disable_txn_auto_retry = 0`. This has the benefit of fewer `COMMIT` statements generating errors.
```